extensions/inkeep/README.md

```diff
@@ -0,0 +1,23 @@
+# Inkeep Raycast Extension
+
+A Raycast extension for interacting with the Inkeep Completions API ([docs](https://docs.inkeep.com/ai-api/chat-completions-api)). This extension allows you to ask questions and get AI-powered answers from your knowledge base.
+
+![Inkeep Raycast Extension](./assets/readme-screenshot.png)
```

Comment:
```suggestion
![Inkeep Raycast Extension](./metadata/readme-screenshot.png)
```

You can remove the one in `assets`

Reply:
Thanks, I did this as the PR template included

> I checked that assets used by the README are placed outside of the metadata folder

Might make sense to update the PR template